igvc_description/urdf/jessi.urdf.xacro

```diff
@@ -0,0 +1,460 @@
+<?xml version="1.0"?>
+
+<robot name="igvc_platform" xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:include filename="$(find velodyne_description)/urdf/VLP-16.urdf.xacro"/>
+
+  <!-- constants -->
+  <xacro:property name="M_PI" value="3.1415926535897931" />
+
+  <!-- wheels -->
+  <xacro:property name="drive_tire_diameter" value="0.305"/>
+  <xacro:property name="drive_tire_width" value="0.0762"/>
+  <!-- TODO -->
+  <xacro:property name="drive_wheel_mass" value="5.0"/>
+
+  <xacro:property name="wheelbase" value="0.714333"/>
+
+  <!-- wheel acuator limits -->
+  <xacro:property name="drive_wheel_eff_limit" value="8"/>
+  <xacro:property name="drive_wheel_vel_limit" value="20"/>
+
+  <!-- drive shock params-->
+  <xacro:property name="drive_wheel_travel" value="0.0"/>
+  <xacro:property name="drive_shock_eff_limit" value="5.0"/>
+  <xacro:property name="drive_shock_vel_limit" value="5.0"/>
+
+  <!-- casters -->
+  <xacro:property name="caster_radius" value="0.1016"/>
+  <!-- TODO -->
+  <xacro:property name="caster_mass" value="1.0"/>
+  <xacro:property name="caster_forward" value="0.612978"/>
+
+  <!-- caster shock params -->
+  <xacro:property name="caster_wheel_travel" value="0.5"/>
+  <xacro:property name="caster_shock_eff_limit" value="200.0"/>
+  <xacro:property name="caster_shock_vel_limit" value="280"/>
+
+  <!-- camera params -->
+  <xacro:property name="camera_y" value="0.0"/>
+  <xacro:property name="camera_x" value="0.216371"/>
+  <xacro:property name="camera_pitch" value="0.0"/>
+  <xacro:property name="camera_height" value="1.354330"/>
```

Comment:
Why do each of these sensor measurements have inconsistent naming? Why x,y,height instead of just x,y,z?